hazelcast-sql/src/test/java/com/hazelcast/jet/sql/impl/connector/kafka/SqlJsonTest.java

```diff
@@ -218,160 +169,120 @@ public void test_allTypes() {
                         LocalDate.of(2020, 4, 15),
                         LocalDateTime.of(2020, 4, 15, 12, 23, 34, 1_000_000),
                         OffsetDateTime.of(2020, 4, 15, 12, 23, 34, 200_000_000, UTC),
-                        ImmutableMap.of("42", 43), // JSON serializer stores maps as JSON objects, the key is converted to a string
+                        Map.of("42", 43), // JSON serializer stores maps as JSON objects, the key is converted to a string
                         null
                 ))
         );
     }
 
     @Test
     public void when_createMappingNoColumns_then_fail() {
-        assertThatThrownBy(() ->
-                sqlService.execute("CREATE MAPPING kafka "
-                        + "TYPE " + KafkaSqlConnector.TYPE_NAME + ' '
-                        + "OPTIONS ('valueFormat'='" + JSON_FLAT_FORMAT + "')"))
+        assertThatThrownBy(() -> kafkaMapping("kafka").create())
```

Comment:
generally the DSL for mappings is very nice, however in such cases when you test validation it is harder to see what is rejected. And also the DSL must allow creating invalid/incomplete objects.

Reply:
The previous version is not invalid or incomplete in terms of parsing. `SqlMapping` and `SqlType` support everything that can be parsed and successfully validated. In this particular example, it is very clear that a Kafka mapping is tried to create without fields, which is rejected.